Projects/Weather app/script.js

```diff
@@ -0,0 +1,28 @@
+var button = document.querySelector(".submit");
+var inputValue = document.querySelector(".search");
+var name = document.querySelector(".city");
+var temp = document.querySelector(".temp");
+var humidity = document.querySelector(".humidity");
+var description = document.querySelector(".description");
+var wind = document.querySelector(".wind");
+
+button.addEventListener("click", function(){
+    fetch('https://api.openweathermap.org/data/2.5/weather?q='+inputValue.value+'&appid=928f9d9a81d8fb865b929fe9d1344bbc')
+    .then(response=>response.json())
+    .then(data=> {
```

Comment:
We can make this a function and use this feature of API calling whenever we want.